lib/vmdb/loggers/io_logger.rb

```diff
@@ -0,0 +1,31 @@
+module Vmdb::Loggers
+  class IoLogger < StringIO
+    def initialize(logger, level = :info, prefix = nil)
+      @logger = logger
+      @level  = level
+      @prefix = prefix
+      super()
+    end
+
+    def write(string)
+      @buffer ||= ""
+      @buffer << string
+      dump_buffer if string.include?("\n")
```

Comment:
@bdunne wouldn't the strip have removed the \n before the write is called. Does that mean all the messages would be buffered till the end because they don't have a \n